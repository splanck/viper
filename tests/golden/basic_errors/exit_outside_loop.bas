10 EXIT WHILE
20 END
