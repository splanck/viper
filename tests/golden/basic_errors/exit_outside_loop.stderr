B1011.*EXIT WHILE used outside of any loop
